src/nwb2bids/_command_line_interface/_main.py

```diff
@@ -92,12 +101,16 @@ def _run_convert_nwb_dataset(
         message = "Please provide at least one NWB file or directory to convert."
         raise ValueError(message)
     handled_nwb_paths = [pathlib.Path(nwb_path) for nwb_path in nwb_paths]
+    handled_sanitization_level = (
+        SanitizationLevel(int(sanitization)) if sanitization.isdigit() else getattr(SanitizationLevel, sanitization)
```

Comment:
```suggestion
        SanitizationLevel(int(sanitization)) if sanitization.isdigit() else SanitizationLevel[sanitization]
```

Just a shortcut syntax.